composeApp/src/commonMain/kotlin/com/developersbreach/kotlindictionarymultiplatform/ui/screens/detail/DetailScreenUI.kt

```diff
@@ -1,29 +1,21 @@
 package com.developersbreach.kotlindictionarymultiplatform.ui.screens.detail
 
 import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.rememberScrollState
-import androidx.compose.foundation.verticalScroll
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.unit.dp
 import com.developersbreach.kotlindictionarymultiplatform.data.detail.model.KotlinTopicDetails
 
 @Composable
 fun DetailScreenUI(topic: KotlinTopicDetails) {
```

Comment:
This object `KotlinTopicDetails` should not be passed down to UI.
Right now your Composables work directly with KotlinTopicDetails, which couples your UI layer to the data layer. You’ll want to introduce a UI model DetailUi and map from the domain object to that UI model in the ViewModel.